### PR TITLE
Update default limits for get_full_accounts API

### PR DESCRIPTION
**Pull request by abitmore**
For issue #2165.

TODO:
- [ ] update release notes

### CODE REVIEW

File: tests/tests/api_limit_tests.cpp [hunk 1]
```diff
@@ -77,45 +77,30 @@ BOOST_AUTO_TEST_CASE( api_limit_get_full_accounts ) {
...
 
+      for( size_t i = 0; i < 51; ++i )
+      {
+         string account_name = "testaccount" + fc::to_string(i);
+         create_account( account_name );
+         accounts.push_back( account_name );
+      }
+
+      // Too many accounts
       GRAPHENE_CHECK_THROW(db_api.get_full_accounts(accounts, false), fc::exception);
```
Comment (abitmore) [inline tests/tests/api_limit_tests.cpp:91]: Missing test cases about the limitations on maximum sizes of returned lists.
